game-core/src/main/java/games/strategy/triplea/delegate/battle/steps/fire/air/AirVsNonSubsStep.java

```diff
@@ -0,0 +1,33 @@
+package games.strategy.triplea.delegate.battle.steps.fire.air;
+
+import games.strategy.engine.data.Unit;
+import games.strategy.engine.delegate.IDelegateBridge;
+import games.strategy.triplea.delegate.ExecutionStack;
+import games.strategy.triplea.delegate.Matches;
+import games.strategy.triplea.delegate.battle.BattleActions;
+import games.strategy.triplea.delegate.battle.BattleState;
+import games.strategy.triplea.delegate.battle.steps.BattleStep;
+import java.util.Collection;
+
+/** Air can not attack subs unless a destroyer is present */
+public abstract class AirVsNonSubsStep extends BattleStep {
+
+  public AirVsNonSubsStep(final BattleState battleState, final BattleActions battleActions) {
+    super(battleState, battleActions);
+  }
+
+  @Override
+  public BattleAtomic getExecutable() {
+    return null;
```

Comment:
nit, if the return value is nullable, at the very least the method should be marked as `@Nullable`. Perhaps better though if this returns `Optional<BattleAtomic>` so that any caller won't mistakenly think the return value is non-nullable and run into a NPE.